Computer Vision/WEAPON-DETECTION-SYSTEM/README.md

```diff
@@ -0,0 +1,97 @@
+Weapon Detection Using YOLOv3
+=============================
+
+Project Overview
+----------------
+
+This project utilizes the YOLOv3 (You Only Look Once) algorithm for real-time weapon detection. The code supports two modes of operation:
+
+1.  **Webcam-based weapon detection**: Uses the webcam to detect weapons in real-time.
+2.  **Video file-based weapon detection**: Uses a pre-recorded video file to detect weapons.
+
+Features
+--------
+
+-   Real-time detection of weapons using either a webcam or a video file.
+-   Display of bounding boxes and labels for detected objects.
+-   Customizable detection thresholds.
+
+Technologies and Libraries Used
+-------------------------------
+
+-   **Python (version 3.x)**
+-   **OpenCV**: Library for computer vision tasks.
+-   **NumPy**: Library for numerical computations.
+-   **YOLOv3**: Pre-trained deep learning model for object detection.
+
+Getting Started
+---------------
+
+### Prerequisites
+
+Make sure you have the following installed:
+
+-   Python 3.x
+-   OpenCV
+-   NumPy
+
+You can install the required libraries using pip:
+
+`pip install opencv-python numpy`
+
+
+
+### Setup
+
+1.  **Clone the repository and navigate to the project directory**:
+
+    ```bash
+    git clone <repository_url>
+    cd <repository_name>
+
+2.  **Download YOLOv3 weights and configuration files**:
```

Comment:
Hey @venkat-2811 ,
Please include the links for the `yolov3.weights` and 'yolov3.cfg'   download.